### ISSUE TITLE
v4.5.3 is not on BootstrapCDN

### ISSUE THREAD

**Issue by mrlife** [twbs/bootstrap#31887]
https://stackpath.bootstrapcdn.com/bootstrap/4.5.3/js/bootstrap.bundle.min.js
https://stackpath.bootstrapcdn.com/bootstrap/4.5.3/css/bootstrap.min.css

Yields error like:

```
<Error>
    <Code>AccessDenied</Code>
    <Message>Access Denied</Message>
    <RequestId>8T8Q3M7J1ZBN1Y8J</RequestId>
    <HostId>
        UoEdHYsY8K4kY7S8P340gWmBZcqUPjlIiE4d5zuHAN+qyA9ZgIOXBsoeSkjGjdQiDygTA3dz9YE=
    </HostId>
</Error>
```

**Comment by jeremygottfried** [comment]
Same

**Comment by mdo** [comment]
CDN hasn't been updated to include the latest, and we might be changing CDNs because of this. Stay tuned!

**Comment by mdo** [comment]
Fixed by #31899 in our live docs. Rest coming in v4.6.0 with a move to jsDelivr.
